shared-modules/tesseract-3.05.01.json

```diff
@@ -0,0 +1,96 @@
+{
+    "name": "tesseract-bundle",
+    "no-autogen": true,
+    "sources": [
+        {
+            "type": "file",
+            "path": "noop-Makefile",
+            "dest-filename": "Makefile"
+        }
+    ],
+    "modules": [
+        {
+            "name": "libleptonica",
+            "buildsystem": "autotools",
+            "config-opts": [
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/DanBloomberg/leptonica/releases/download/1.74.4/leptonica-1.74.4.tar.gz",
+                    "sha256": "29c35426a416bf454413c6fec24c24a0b633e26144a17e98351b6dffaa4a833b"
+                }
+            ]
+        },
+        {
+            "name": "tesseract",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--disable-graphics"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tesseract-ocr/tesseract/archive/3.05.01.tar.gz",
+                    "sha256": "05898f93c5d057fada49b9a116fc86ad9310ff1726a0f499c3e5211b3af47ec1"
+                }
+            ]
+        },
+        {
+            "name": "tessdata",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tesseract-ocr/tessdata/archive/3.04.00.tar.gz",
+                    "sha256": "5dcb37198336b6953843b461ee535df1401b41008d550fc9e43d0edabca7adb1"
+                },
+                {
+                    "type": "file",
+                    "path": "noop-Makefile",
+                    "dest-filename": "Makefile"
+                }
+            ],
+            "post-install": [
+                "mkdir -p /app/share/tessdata",
+                "cp eng.traineddata /app/share/tessdata",
+                "cp osd.traineddata /app/share/tessdata",
+
+                "mkdir -p /app/share/locale/bg",
```

Comment:
Nitpick: you can use regular bash constructs here, like `for` loop

Reply:
Good point. That's something I will do next time I add a language.